packages/admin/admin-generator/src/commands/generate/generateGrid/generateGrid.ts

```diff
@@ -884,6 +886,7 @@ export function generateGrid<T extends { __typename?: string }>(
                         : ""
                 }
         ];
+        },[intl${gridNeedsTheme ? ", theme" : ""}${showCrudContextMenuInActionsColumn ? ", client" : ""}]);
```

Comment:
When using rowAction it's crucial that this also must be wrapped inside a useCallback when it will be passed as a prop.

Reply:
TODO: we also need to add rowAction as a dependency here